CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/mysql_util.py

```diff
@@ -73,7 +76,8 @@ def get_gtid_mode_on(self):
     return row[0] == 'ON'
 
   def get_views(self):
-    """ Returns a list of views on the database
+    """ Returns a list of views on the database.
```

Comment:
Curious, why do we need the next "Returns:" section if this line is exactly the same?